packages/python/src/version.ts

```diff
@@ -42,7 +56,16 @@ export function getLatestPythonVersion({
   if (isDev) {
     return getDevPythonVersion();
   }
```

Comment:
> This way local dev with vc dev will work for users with newer installs and existing environments 

The PR description says this but `vc dev` should be hitting the code below, it should hit the `isDev` code and call `getDevPythonVersion()`

Reply:
I stand corrected. `vc dev` is fine. `vc build` is the issue:

```
$ vc build
Vercel CLI 32.4.1
Installing required dependencies...
Failed to run "pip3.9 install --disable-pip-version-check --target . werkzeug==1.0.1"
Error: spawn pip3.9 ENOENT
```

Reply:
Can you update the PR description?